app/src/main/java/com/example/fizzbuzz/fizzbuzz.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.fizzbuzz
+
+fun main() {
+    fizzbuzz(100)
+}
+
+fun fizzbuzz(maxNumber: Int) {
```

Comment:
We have a function here that never has a return value. While usually fine, could point to issues with code smell or just not being extensible. 
It's fine to leave it as is for now, but consider if this will be extensible in to part 2.